runtime/services/catalog/migrations.go

```diff
@@ -626,14 +646,21 @@ func (s *Service) runMigrationItems(
 // TODO: should we remove from dag if validation fails?
 // TODO: store only valid metrics view
 
-func (s *Service) createInStore(ctx context.Context, item *MigrationItem) error {
```

Comment:
Actually errors from these methods are handled and added to the result anyway. So we can just return the errors from these.